src/elements/stepper/stepper/stepper.stories.ts

```diff
@@ -264,7 +267,9 @@ const WithMultipleFormsTemplate = (args: Args): TemplateResult => {
 
       <sbb-step-label icon-name="tick-small">Step 4</sbb-step-label>
       <sbb-step>
-        <div style="margin-block-end: var(--sbb-spacing-fixed-4x)">You are now done.</div>
+        <div style="margin-block-end: var(--sbb-spacing-fixed-4x)" aria-live="polite">
```

Comment:
Does that really work as it is always present?

Reply:
It works at least the first time you land on the last step, but I didn't test it on every screen reader. I will do it with the next generated link.

Reply:
Yes, it works, I tested it with NVDA and JAWS as well.

Reply:
ok nice. so aria live does not when it's hidden and later presented.